### ISSUE TITLE
Drop progress column from policy table - after #388 is live for a bit

### ISSUE THREAD

**Issue by FollowSteph** [PathmindAI/pathmind-webapp#692]
I'm putting this as a separate issue to #388 for safety reasons. The progress (json) column is no longer used after #388 is live but I've kept it to make rolling back easier and safer as a separate issue. Even if we make database backups it's a lot easier to quickly revert the webapp version than revert the database.
